contracts/cw721-base/src/query.rs

```diff
@@ -207,6 +207,10 @@ where
             },
         })
     }
+
+    fn ownership(deps: Deps) -> StdResult<cw_ownable::Ownership<Addr>> {
```

Comment:
I think we should return `Ownership<String>` here. The idea is that contract APIs (i.e. messages and responses) should only contain unchecked types.

Reply:
This probably doesn't matter anyways, since the results is serialized into JSON